packages/fax/src/models/v3/fax/fax.ts

```diff
@@ -0,0 +1,62 @@
+import { BarCode } from '../bar-code';
+import { FaxContentUrl } from '../fax-content-url';
+import { FaxMoney } from '../fax-money';
+import { ErrorType, FaxDirection, FaxStatus, ImageConversionMethod, WebhookContentType } from '../enums';
+
+export interface Fax {
+
+  /** The id of a fax */
+  id?: string;
```

Comment:
Is it necessary to have all fields within this interface marked as optional?
`id` seems to be like not an optional thing.

Reply:
This is what the specifications says